docs/source/federation/introduction.mdx

```diff
@@ -6,23 +6,11 @@ description: Implement a single data graph across multiple services
 
 import FederationArchitecture from '../../shared/diagrams/federation-architecture.mdx';
 
-To get the most out of GraphQL, your organization should expose a [single data
-graph](https://principledgraphql.com/integrity#1-one-graph) that provides a
-unified interface for querying all of your backing data sources. This
-allows clients to fetch data from any number of sources simultaneously,
-_without_ needing to know which data comes from which source.
-
-As your data graph grows, however, it can become inefficient or even difficult
-to represent the graph with a single, monolithic GraphQL server. To remedy this,
-you can divide your data graph's implementation across multiple composable
-services with **Apollo Federation**.
-
-Unlike other distributed GraphQL
-architectures (such as schema stitching), Apollo Federation uses a declarative
-programming model that enables each **implementing service** to
-implement _only_ the part of your data graph that it should be responsible for.
-This way, your organization can represent an enterprise-scale data graph as a
-collection of separately maintained GraphQL services.
+To get the most out of GraphQL, your organization should expose a [single data graph](https://principledgraphql.com/integrity#1-one-graph) that provides a unified interface for querying any combination of your backing data sources. However, it can be challenging to represent an enterprise-scale data graph with a single, monolithic GraphQL server. To remedy this, you can divide your graph's implementation across multiple composable services with **Apollo Federation**.
+
+Unlike other distributed GraphQL architectures (such as schema stitching), Apollo Federation uses a declarative programming model that enables each **implementing service** to implement _only_ the part of your graph that it's responsible for.
+
+Apollo Federation also supports a [managed mode with Apollo Studio](https://www.apollographql.com/docs/studio/managed-federation/overview/), which enables you to add, remove, and refactor your implementing services without requiring any downtime for your production graph. Managed federation is a free Studio feature that does not require a paid plan.
```

Comment:
I think that a lot of people will read the first sentence of the paragraph and skip it because it seems like we're trying to sell something. I know I'm being a little nitpicky, but could we try to make this clear in the first sentence?

Reply:
Done, was initially concerned it would look like "free-managed federation" is a thing, but I think that's clear enough with the link's coverage